js.variables/variables.js

```diff
@@ -0,0 +1,17 @@
+var adminName = "Kate";
+var userName = "Black";
+
+console.log(typeof adminName); // string
+console.log(typeof userName);  // string
+
+var message = "awesome"
+
+var password1 = 1234;
+var password2 = 1111;
+console.log(`Javascript is: ${message}, ${password1}, ${password2}`);
+
+var Mynumber = "23";
```

Comment:
for naming better to use camelCase "myNumber"
```suggestion
var myNumber = "23";
```